src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -24,7 +24,6 @@ CA1836 | Performance | Info | PreferIsEmptyOverCountAnalyzer, [Documentation](ht
 CA1837 | Performance | Info | UseEnvironmentProcessId, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1837)
 CA1838 | Performance | Hidden | AvoidStringBuilderPInvokeParametersAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1838)
 CA2011 | Reliability | Info | AvoidInfiniteRecursion, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2011)
-CA2012 | Reliability | Hidden | UseValueTasksCorrectlyAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2012)
```

Comment:
I had to remove this line, because if I leave it there, I get this error:

```
Build FAILED.

AnalyzerReleases.Unshipped.md(52,1): error RS2005: Rule 'CA2012' has more then one entry for release 'unshipped' in analyzer release file 'AnalyzerReleases.Unshipped.md'. [D:\roslyn-analyzers\src\ 
NetAnalyzers\Core\Microsoft.CodeAnalysis.NetAnalyzers.csproj]
    0 Warning(s)
    1 Error(s)

```